src/main/java/ua/bookstore/online/service/BookServiceImpl.java

```diff
@@ -61,6 +63,9 @@ public BookDto update(Long id, CreateBookRequestDto bookRequestDto) {
 
     @Override
     public void delete(Long id) {
-        bookRepository.deleteById(id);
+        if (bookRepository.existsById(id)) {
```

Comment:
I'd flip that `if`, to follow the fail early pattern

Reply:
But I think my it is better than `!` at the beginning.
```suggestion
        if (!bookRepository.existsById(id)) {
```
Or should I write it like :
```suggestion
        if (bookRepository.existsById(id) == false) {
```